modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -75,12 +75,13 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 $ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
 ----
 
-. Create a new `MachineConfig` object for master or worker and add the `journal.conf` parameters:
+. Create a new `MachineConfig` object including the encoded contents of `journald.conf` (jrnl_cnf variable):
```

Comment:
"Create a new" is redundant. 

I think that jrnl_cnf should be a codephrase `jrnl_cnf`. 

Since we know what the encode contents of the variable are, because we made it in the last step, I think we should change the sentence structure around rather than mention jrnl_cnf variable parenthetically at the end. 

What about:

>Create a `MachineConfig object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the last step.

I'm not sure this is exactly it, but consider moving `jrnl_cnf` toward the middle of the sentence to provide more context about what it is.

Reply:
Your wording seems good - i've made that change.